demo/android/rust/src/lib.rs

```diff
@@ -167,6 +170,28 @@ impl Window for WindowImpl {
         Point2DI32::new(0, 0)
     }
 
+    fn view_box_size(&self, mode: Mode) -> Point2DI32 {
+        let mut width = self.size.x();
+        let height = self.size.y();
+        if let Mode::VR = mode {
+            width = width / 2;
+        }
+        Point2DI32::new(width as i32, height as i32)
+    }
+
+    fn make_current(&mut self, mode: Mode, index: Option<u32>) -> RectI32 {
```

Comment:
I think this is a bit confusing for programmers who are familiar with OpenGL. `glxMakeCurrent`, `eglMakeCurrent`, etc. just make the context current without setting the viewport. How about just having another method `viewport` that subsumes `view_box_size` and the size calculation of this method? That will also avoid a bit of code duplication (the halving of `width`).